d3a_api_client/commands.py

```diff
@@ -16,12 +19,18 @@ def offer_energy(self, area_uuid, energy, price):
     def offer_energy_rate(self, area_uuid, energy, rate):
         return self._add_to_buffer(area_uuid, Commands.OFFER, {"energy": energy, "rate": rate})
```

Comment:
I recall a discussion we had some PRs ago, I think that d3a API does not support the "rate" parameter. Therefore this needs to change:
```suggestion
        return self._add_to_buffer(area_uuid, Commands.OFFER, {"energy": energy, "price": rate * energy})
```

Reply:
@spyrostz the mentioned PR is not merged yet, it is fixed there so they should sync upon merging

Reply:
That works for me, however make sure that they are not using this in the researcher's scripts (I am mentioning because IIRC they usually use the energy rate, and if you do not apply this fix their scripts will fail). If they do not use it then feel free to merge this PR, otherwise just port the fix in this PR (IIRC it is only 2 these lines that you need to adapt)

Reply:
Done